### DISCUSSION TITLE
Watcher webhooks

### DISCUSSION THREAD

**Discussion by rezatakhshid** [BlueBrain/nexus#3718] [Q&A] [answered]
Hi,
Is it possible to setup watcher webhooks on Delta?
Let's say I want to send notification to users if there is a new version of a project or dataset has become available. 
Thank you

**Comment by imsdu** [comment, marked as answer]
Hello,
Currently it is only possible to fetch server side events at the project level and it only returns the Delta events which may not be what you are looking for: https://bluebrainnexus.io/docs/delta/api/projects-api.html#server-sent-events

What I understand is you are want to target a particular entity or maybe a set of entities matching a given condition ?
What would you expect to be returned ? The same result as the fetch operation ?

**Reply by rezatakhshid** [reply]
@imsdu Thanks for your response.
I'm looking for something along the lines of this:
https://www.elastic.co/guide/en/elasticsearch/reference/current/actions-webhook.html

so we can setup webhooks and define events. 
specific event I'm looking for is if there is a new version uploaded for a dataset. 
so the flow would be:
new version is uploaded > the trigger is activated > endpoint is called > (desired tasks are done from here on. for example notify relevant people)

**Reply by imsdu** [reply]
Ok, understood.
We will discuss this internally as there has been needs similar to these needs expressed and see if you can include it in the roadmap.

**Reply by rezatakhshid** [reply]
Awesome. We could probably get by, setting scheduled tasks for now, but looking forward to see this feature getting released. Thanks.
